tfchain_bridge/pkg/bridge/bridge.go

```diff
@@ -278,6 +278,19 @@ func (bridge *Bridge) mint(senders map[string]*big.Int, tx hProtocol.Transaction
 		depositedAmount = amount
 	}
 
+	if tx.MemoType == "hash" {
```

Comment:
There is both a "hash" and "return" memo type, and they both have 32 bytes value. So it seems that this either ignores too much or the wrong thing?